app/controllers/comments_controller.rb

```diff
@@ -1,5 +1,9 @@
 class CommentsController < ApplicationController
   # create, update, destroy
+  def new
```

Comment:
#new method is never used. There is no need to render a page for new comment, since comments are already shown inside the event show page.
There is no route to call comments#new method, and is not needed anyway.